File: Java Barcode Scanner ME/app/src/main/java/de/dimedis/mobileentry/fragments/util/DecoderBarCode.java
package de.dimedis.mobileentry.fragments.util;

import java.util.StringTokenizer;

public class DecoderBarCode {

    protected String codeA;
    protected String codeB;

    public DecoderBarCode(String code) {
        StringTokenizer st = new StringTokenizer(code);
        if (st.hasMoreTokens()) codeA = st.nextToken();
        if (st.hasMoreTokens()) codeB = st.nextToken();
    }
}

File: Kotlin Migration/NTMobileEntry/app/src/main/java/de/dimedis/mobileentry/fragments/util/DecoderBarCode.kt
package de.dimedis.mobileentry.fragments.util

import java.util.*
/**
 * Created by Softeq Development Corporation
 * http://www.softeq.com
 */
open class DecoderBarCode(code: String?) {
    protected var codeA: String? = null
    protected var codeB: String? = null

    init {
        val st = StringTokenizer(code)
        if (st.hasMoreTokens()) codeA = st.nextToken()
        if (st.hasMoreTokens()) codeB = st.nextToken()
    }
}